pkg/platform/kube/test/platform_test.go

```diff
@@ -1698,25 +1699,62 @@ def handler(context, event):
 	suite.DeployFunctionAndRedeploy(createFunctionOptions, afterFirstDeploy, afterSecondDeploy)
 }
 
-func (suite *DeleteFunctionTestSuite) TestProcessorShutdown() {
+func (suite *DeployFunctionTestSuite) TestProcessorShutdown() {
 	functionName := "func-to-test-processor-shutdown"
 	createFunctionOptions := suite.CompileCreateFunctionOptions(functionName)
 
+	// change source code
+	createFunctionOptions.FunctionConfig.Spec.Build.FunctionSourceCode = base64.StdEncoding.EncodeToString([]byte(`
+terminated = False
+def handler(context, event):
+    if terminated:
+        context.logger.info("Got event after termination")
+    else:
+        context.logger.info("Got http event")
+    return "got http event!"
+
+def callback():
+    import time
+    time.sleep(3)
+    terminated = True
+
+def init_context(context):
+    context.platform.set_termination_callback(callback)
+`))
```

Comment:
suggestion -
why not using go embed? would let you write python in `x.py` and have it linted and etc

Reply:
@liranbg That's a good idea. I can implement it in another PR and transfer all our raw-code function definitions in tests to embed, just to maintain a holistic approach. What do you think?